documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -223,11 +223,24 @@ There are also other options available
 
 * `io.debezium.relational.history.FileDatabaseHistory` for non-Kafka deployments
 * `io.debezium.relational.history.MemoryDatabaseHistory` volatile store for test environments
+* `io.debezium.server.redis.RedisDatabaseHistory` volatile store for test environments
 
 |[[debezium-source-database-history-file-filename]]<<debezium-source-database-history-file-filename, `debezium.source.database.history.file.filename`>>
 |
 |The name and location of the file to which `FileDatabaseHistory` persists its data.
 
+|[[debezium-source-database-history-redis-address]]<<debezium-source-database-history-redis-address, `debezium.source.database.history.redis.address`>>
+|
+|The Redis host:port to connect to if using `RedisDatabaseHistory`.
+
+|[[debezium-source-database-history-redis-password]]<<debezium-source-database-history-redis-password, `debezium.source.database.history.redis.password`>>
```

Comment:
User property seems missing?

Reply:
Also those retry ones; or did you omit them on purpose?

Reply:
done